app/views/shared/_footer_lite.html.erb

```diff
@@ -47,14 +47,14 @@
       </div>
       <div class="display-flex flex-align-center flex-justify-center">
         <% if show_language_dropdown %>
-          <ul class='list-reset display-none tablet:display-block margin-bottom-0'>
-            <li class="i18n-desktop-toggle display-flex margin-y-1 margin-x-2">
+          <ul class='usa-list--unstyled display-none tablet:display-block padding-y-1 padding-x-2'>
```

Comment:
I tested this again after the latest changes and unfortunately it's still not working quite right, specifically due to [extra padding applied when the button is toggled open](https://github.com/18F/identity-idp/blob/b95fabd1ea96480ebfb8de5150598f1f6115fa27/app/assets/stylesheets/components/_i18n-dropdown.scss#L35-L38).

Stepping back, I wonder why this is a list in the first place, because it's only containing the one item for the button. What if we just made it a `div` so that we can avoid the peculiarities of the unstyled list?

Tinkering locally, something like this seems to keep the current styles and simplify a little:

```diff
diff --git a/app/assets/stylesheets/components/_i18n-dropdown.scss b/app/assets/stylesheets/components/_i18n-dropdown.scss
index d54bb529a..4c35bae96 100644
--- a/app/assets/stylesheets/components/_i18n-dropdown.scss
+++ b/app/assets/stylesheets/components/_i18n-dropdown.scss
@@ -31,11 +31,6 @@
     border-radius: 0;
     margin-bottom: 0;
     margin-top: 0;
-
-    button {
-      padding-bottom: 12px;
-      padding-top: 12px;
-    }
   }
 }
 
diff --git a/app/views/shared/_footer_lite.html.erb b/app/views/shared/_footer_lite.html.erb
index 1f8c26c6c..b36bf6e58 100644
--- a/app/views/shared/_footer_lite.html.erb
+++ b/app/views/shared/_footer_lite.html.erb
@@ -47,25 +47,23 @@
       </div>
       <div class="display-flex flex-align-center flex-justify-center">
         <% if show_language_dropdown %>
-          <ul class='usa-list--unstyled display-none tablet:display-block padding-y-1 padding-x-2'>
-            <li class="i18n-desktop-toggle display-flex">
-              <button class='text-white text-decoration-none border border-primary radius-lg padding-x-1 py-tiny language-desktop-button' aria-expanded='false'>
-                <%= image_tag asset_url('globe-white.svg'), width: 12,
-                                                            class: 'margin-right-1', alt: '' %><%= t('i18n.language') %>
-                <span class="caret display-inline-block margin-left-05" aria-hidden="true">&#9662;</span>
-              </button>
-              <ul class="i18n-desktop-dropdown usa-list--unstyled margin-bottom-0 text-white display-none">
-                <% I18n.available_locales.each do |locale| %>
-                  <li class='border-bottom border-primary-darker'>
-                    <%= link_to t("i18n.locale.#{locale}"),
-                                sanitized_requested_url.merge(locale: locale),
-                                class: 'display-block pl-24p padding-y-2 text-decoration-none text-white',
-                                lang: locale == I18n.locale ? nil : locale %>
-                  </li>
-                <% end %>
-              </ul>
-            </li>
-          </ul>
+          <div class='i18n-desktop-toggle display-none tablet:display-flex margin-x-2 padding-y-1'>
+            <button class='text-white text-decoration-none border border-primary radius-lg padding-x-1 py-tiny language-desktop-button' aria-expanded='false'>
+              <%= image_tag asset_url('globe-white.svg'), width: 12,
+                                                          class: 'margin-right-1', alt: '' %><%= t('i18n.language') %>
+              <span class="caret display-inline-block margin-left-05" aria-hidden="true">&#9662;</span>
+            </button>
+            <ul class="i18n-desktop-dropdown usa-list--unstyled margin-bottom-0 text-white display-none">
+              <% I18n.available_locales.each do |locale| %>
+                <li class='border-bottom border-primary-darker'>
+                  <%= link_to t("i18n.locale.#{locale}"),
+                              sanitized_requested_url.merge(locale: locale),
+                              class: 'display-block pl-24p padding-y-2 text-decoration-none text-white',
+                              lang: locale == I18n.locale ? nil : locale %>
+                </li>
+              <% end %>
+            </ul>
+          </div>
         <% end %>
 
         <div class="display-flex tablet:display-none">
```

Reply:
Applied in 4147c62